src/main/resources/api.yaml

```diff
@@ -0,0 +1,308 @@
+            type: string
+      responses:
+        '200':
+          description: Details of a single Quiz
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/QuizDetails'
+  /quizzes/details:
+    post:
+      summary: Create a quiz details
+      operationId: createQuizDetails
+      tags:
+        - quizzes
+      requestBody:
+        description: QuizDetails to create
+        required: true
+        content:
+          application/json:
+            schema:
+              $ref: '#/components/schemas/QuizDetails'
+      responses:
+        '201':
+          description: QuizDetails created
+          content:
+            application/json:
+              schema:
+                type: array
+                items:
+                  $ref: '#/components/schemas/QuizDetails'
+  /questions:
+    post:
+      summary: Create questions
+      operationId: createQuestions
+      tags:
+        - questions
+      requestBody:
+        description: Questions to create
+        required: true
+        content:
+          application/json:
+            schema:
+              type: array
+              items:
+                $ref: '#/components/schemas/Question'
+      responses:
+        '201':
+          description: Questions created
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/Question'
+  /questions/{quizId]:
+    get:
+      summary: Questions for quiz
+      operationId: showQuestionsByQuizId
+      tags:
+        - questions
+      parameters:
+        - name: quizId
+          in: path
+          required: true
+          schema:
+            type: string
+      responses:
+        '200':
+          description: Questions of a single Quiz
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/Question'
+components:
+  schemas:
+    Note:
+      type: object
+      required:
+        - id
+        - content
+      properties:
+        id:
+          type: integer
+          format: int64
+        content:
+          type: string
+    Quiz:
+      type: object
+      required:
+        - id
+        - workspaceId
+        - title
+        - score
+      #        - questions
+      properties:
+        id:
+          type: string
+        workspaceId:
+          type: string
+        title:
+          type: string
+        score:
+          type: string
+        author:
+          type: string
+        date:
+          type: string
+    QuizSimpleResult:
+      type: object
+      required:
+        - incorrect
+        - correct
+      properties:
+        incorrect:
+          type: integer
+          format: int32
+        correct:
+          type: integer
+          format: int32
+    QuizDetails:
+      type: object
+      required:
+        - id
+        - name
+        - description
+        - numberOfExercises
+        - lastScore
```

Comment:
I just realised we need to store scores on backend xD - on start we can just add simple endpoint with correct/incorrect numbers in request body, but in future we probably would like to store more details - to create attempt dedicated for user

Reply:
sure